addon/mixins/ajax-request.js

```diff
@@ -222,7 +229,7 @@ export default Mixin.create({
       url: hash.url
     };
 
-    if (isJSONAPIContentType(hash.headers['Content-Type']) && requestData.type !== 'GET') {
+    if (isJSONAPIContentType(hash) && requestData.type !== 'GET') {
```

Comment:
I personally think it would make more sense to leave the `isJSONAPIContentType` function alone and instead just use it twice, something like

```javascript
if ((isJSONAPIContentType(hash.contentType) || isJSONAPIContentType(hash.headers['Content-Type']) && request.type !== 'GET') {
  ...
}
```

if we need to check both properties.